README.md

```diff
@@ -25,3 +25,65 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
+
+<hr style="height:0.3px;border:none;color:lightgrey;background-color:lightgrey;" />
+
+<p align="center">
+<a href="mailto:sales@syncfusion.com?Subject=Syncfusion Angular Accordion - GitHub" target="_top">sales@syncfusion.com</a> | <a href="https://www.syncfusion.com?utm_source=github&utm_medium=listing&utm_campaign=angular-accordion-github-samples)">www.syncfusion.com</a> | Toll Free: 1-888-9 DOTNET <br>
+</p>
+Footer
```

Comment:
is this needed?
<img width="193" alt="image" src="https://github.com/SyncfusionExamples/ej2-angular-11-accordion/assets/61148953/e45654ba-b797-4dc3-93c4-fd232511ee9b">